### ISSUE TITLE
Connect without loading a saved config

### ISSUE THREAD

**Issue by sidvishnoi** [SchoofsKelvin/vscode-sshfs#56]
It would be a nice feature if we can connect without saving a configuration first and prompt user for a dynamic config as:
``` 
> prompt
username:password@host:port/directory
// (or something similar)
```

**Comment by SchoofsKelvin** [comment]
I'm not sure how easily this is do accomplish:
- When you "connect" a SSH FS, I basically add a new workspace folder with `ssh://<name>` as URI
- VSCode automatically reloads the window when the main/first workspace changes
- After the potential reload, it'll try to load the workspace, sees it's ssh://<name>, and ask my extension
- I extract the <name> from the URI, get the right configuration for it, and use that to connect

For "temporary" sessions, I'd have to write the configs somewhere, or hope/assume the window won't get reloaded as long as the workspace folder is used. I'll look into it once I have a bit more time, though.

**Comment by SchoofsKelvin** [comment]
Seems like I kept another issue updated but forgot updating this issue too:
This feature has been added a while back (844b0e1) and is already available since v1.19.1 of the extension.

It pretty much supports the syntax you suggested, as it's basically the standard `ssh://`/`sftp://` syntax. It (currently) doesn't support configuring anything extra, but:
- It doesn't support the `:password` part. Unless you're "inheriting" from an existing config (see below), it'll just prompt for it
- In the future, I might add e.g. `username;someProperty=someValue,etc=etc@...` although not anytime soon
- It does support using existing configs and even existing PuTTY sessions
  e.g. `anotherUser@someConfig` should work just fine, as should `SomeSessionName/path`
- Issue #107 is about adding support for SSH config files (such as `.ssh/config`), which can also be referenced like the PuTTY

These all mount "specially formatted" URIs as workspace folder to your workspace. It's basically just `ssh://<input>/` e.g. `ssh://username@host:port/some/subdirectory`, with the only two exceptions being that it _always_ adds a `@`, so without a username you have e.g. `ssh://@host:port/...`, and that if you have a config name containing `@`, instant connections might not really play nice with it....
